### ISSUE TITLE
No screen reader feedback when ingesting a new document

### ISSUE THREAD

**Issue by sashathomas** [ManifoldScholar/manifold#2743]
When ingesting a new text (such as a .epub, or Google Doc) in the "Add a New Text" pane, there is no audio feedback that the process is happening, nor is there verbal confirmation when it has completed.

## Expected Behavior
A user should be given audio confirmation that the ingestion process has finished. As most users are going to be uploading some sort of text when creating a new project, it's important that these steps are as clear as possible. In addition to adding audio confirmation, there could also be audio feedback about the ingestion process. As visual users are shown the progress of the ingestion process in a log, non-visual users should get updates about this process as well.

## Current Behavior
When ingesting a new document, there is no audio feedback during the process, or after it has been completed. Similar to the issue about the lack of notification for saving a project (see #2559), there is a progress bar on the screen, but this does not announce to non-visual users, and does not show that the process has been completed. For visual users, there is a “Completed” button after the process is finished, but this is not announced.

## Possible Solution
Adding audio confirmation that a document was successfully uploaded is a good solution. It may be a good idea to add the `aria-live="polite"` attribute to the element that holds this log of events (i.e. `<div class="ingestion-output__log-value" aria-live="polite"`>).

## Steps to Reproduce
1. Navigate to admin interface (through Enter Admin Mode)
2. Navigate to an editable project
3. Navigate to the "texts" panel
4. With a screen reader active, upload a new text of any type
5. Note that there is no feedback during this ingestion process, nor is there confirmation when it has completed

## Context (Environment)
NVDA version 2020.1
Windows 10 Home Version 1909 18363.778
Firefox ESR 68.8.0 ESR

**Comment by zdavis** [comment]
@bot bug

**Comment by zdavis** [comment]
Thank you for taking the time to report this bug. We've reviewed the issue and agree that this should be fixed. We’re adding a “planned” label to indicate that we consider this bug as part of our current work plan.

_This was an automated message, but please don't hesitate to reply. Our team watches these issues closely and will respond as soon as we're able to!_

**Comment by dananjohnson** [comment]
Thanks for flagging this issue, @sashathomas! I opted to make the drawer header into a live region rather than the log ([a54bed8](https://github.com/ManifoldScholar/manifold/pull/3079/commits/a54bed8b03d6408615cbdfc583590d42b7529032)). In testing, I found the log output to be extremely verbose to the point of being unhelpful. The header, by contrast, provides succinct updates about the state of the ingestion: "sleeping," "processing," "finished", etc. Though brief, this seemed like more meaningful feedback for those using a screen reader with Manifold.

<img width="472" alt="Screen Shot 2021-09-08 at 3 37 20 PM" src="https://user-images.githubusercontent.com/12454203/132595250-edbf37cf-1a9f-4b2c-8042-97b68fe804d9.png">
